src/main/kotlin/no/nav/familie/ef/sak/opplysninger/personopplysninger/GrunnlagsdataRegisterService.kt

```diff
@@ -40,7 +41,14 @@ class GrunnlagsdataRegisterService(
             arbeidsforholdService.finnesAvsluttetArbeidsforholdSisteAntallMåneder(
                 grunnlagsdataFraPdl.gjeldendeIdentForSøker(),
             )
-        val harKontantstøttePerioder = kontantstøtteService.finnesKontantstøtteUtbetalingerPåBruker(personIdent).finnesUtbetaling
+        val kontantstøttePerioder = kontantstøtteService.hentUtbetalingsinfoKontantstøtte(personIdent)
+        val harKontantstøttePerioder = kontantstøttePerioder.finnesUtbetaling
+        val kontantstøttePerioderDomene =
+            kontantstøttePerioder
+                .takeIf { harKontantstøttePerioder }
+                ?.let {
+                    it.perioder.map { periode -> KontantstøttePeriode(periode.årMånedFra, periode.årMånedTil, periode.kilde) }
+                }?.sortedByDescending { it.fomMåned }
```

Comment:
Trekk ut dette i en egen funksjon

Reply:
Jeg ville også gjort slik at `kontantstøttePerioderDomene` settes til en tom liste i stedet for `null` når man ikke har kontantstøtteperioder. Da slipper man å ha nullable verdier i ganske mange data klasser i både backend og frontend